pylint/checkers/logging.py

```diff
@@ -174,13 +174,26 @@ def _check_log_method(self, node, name):
         else:
             return
 
-        if isinstance(node.args[format_pos], astroid.BinOp) and node.args[format_pos].op == '%':
-            self.add_message('logging-not-lazy', node=node)
+        if isinstance(node.args[format_pos], astroid.BinOp):
+            binop = node.args[format_pos]
+            if (binop.op == '%' or binop.op == '+' and
+                    len([_operand for _operand in (binop.left, binop.right)
+                         if self._is_operand_literal_str(_operand)]) == 1):
+                self.add_message('logging-not-lazy', node=node)
         elif isinstance(node.args[format_pos], astroid.Call):
             self._check_call_func(node.args[format_pos])
         elif isinstance(node.args[format_pos], astroid.Const):
             self._check_format_string(node, format_pos)
 
+    @staticmethod
+    def _is_operand_literal_str(operand):
+        """
+        Return True if the operand in argument is a literal string
+        """
+        if isinstance(operand, astroid.Const) and operand.name == 'str':
```

Comment:
``return isinstance(operand, astroid.Const) and operand.name == 'str'`` should be enough.